simln-lib/src/lib.rs

```diff
@@ -327,11 +335,21 @@ pub trait DestinationGenerator: Send {
 #[error("Payment generation error: {0}")]
 pub struct PaymentGenerationError(String);
 
+/// A trait for generating payment parameters in the Lightning Network.
+/// 
+/// This trait is responsible for determining when and how payments should be made,
+/// including timing, amounts, and frequency. It can be used to implement various
+/// payment strategies, from simple periodic payments to more complex patterns
+/// based on network conditions or business logic.
 pub trait PaymentGenerator: Display + Send {
     /// Returns the time that the payments should start
     fn payment_start(&self) -> Option<Duration>;
 
-    /// Returns the number of payments that should be made
+    /// Returns the number of payments that should be made.
```

Comment:
Make this comment a little clearer that it should be `Some` if there's a limit on the number of payments we want to dispatch and `None` otherwise.